HETG_CHECK/HETG_Check.py

```diff
@@ -0,0 +1,283 @@
+
+deadman_parser.add_argument("nlet_file_path", help="Path to the NLET file to be used in assembling the history. e.g. /data/acis/LoadReviews/TEST_NLET_FILES/FEB2122A_NonLoadTrackedEvents.txt")
+
+args = deadman_parser.parse_args()
+
+#
+# Inits
+# 
+# Create an instance of the Backstop History class
+BSC = BackstopHistory.Backstop_History_Class('ACIS-Continuity.txt',
+                                             args.nlet_file_path,
+                                             args.review_load_path,
+                                             0)
+# Create an instance of the ORP File Class
+ofci =  ofc.ORP_File_Class()
+
+load_week_path = args.review_load_path
+nlet_file_path = args.nlet_file_path
+
+# Extract the load week out from the path
+load_week = load_week_path.split('/')[5]
+
+# Read the review load - results are in BSC.master_list
+assembled_commands = BSC.Read_Review_Load(BSC.outdir)
+
+# Capture the start date and time of the Review load
+rev_start_date = assembled_commands[0]['date']
+rev_start_time = assembled_commands[0]['time']
+
+# Next, find the status of the HETG at the end of the Continuity load
+cont_status = Get_OFLS_Status_Line(load_week_path)
+
+# Iniitalize the event dates
+if cont_status['HETG_status'] == 'HETG-IN':
+
+    HETG_in_date = cont_status['date']
+    HETG_in_time = apt.secs(HETG_in_date)
+    hetg_in = True
+    
+    HETG_out_date = None
+    HETG_out_time = None
+    hetg_out = False
+    
+    HETG_status = 'IN'
+    # Inform the user that the first results are from the Continuity load.
+    print('\nThis is the Continuity Load:')
+    print('     ' + cont_status['date']+'      HETG INSERTED in the Continuity load')
+else:
+
+    HETG_in_date=None
+    HETG_in_time=None
+    hetg_in = False
+    
+    HETG_out_date=cont_status['date']
+    HETG_out_time=apt.secs(HETG_out_date)
+    hetg_out = True
+    
+    HETG_status = 'UNK'
+
+EQF013M_date = None
+eqf013m_acq = False
+
+EEF1000_date = None
+eef1000_acq = False
+
+XQF013M_date = None
+xqf013m_acq = False
+
+XEF1000_date = None
+xef1000_acq = False
+
+OORMPDS_date = None
+OORMPDS_time = None
+oormpds_acq = False
+
+OORMPEN_date = None
+OORMPEN_time = None
+oormpen_acq = False
+
+# Tell the user what load week we are checking
+print('\n HETG Check for load week: ', load_week)
+
+# Initialize the flag which will indicate when the review load output has begun
+review_load_started_flag = False
+
+# Walk through the assembled history of commands and set dates and flags whenever
+# you read a command or event that we need to note.
+for each_cmd in assembled_commands:
+
+    # If the date of this command is after the Review Load ToFC, then 
+    # print a line out that delineates Continuity load HETG reports from
+    # Review load HETG reports
+    if (each_cmd['time'] >= rev_start_time) and \
+       (review_load_started_flag == False):
```

Comment:
Minor nit, could we do `not review_load_started_flag` here?

Reply:
Changed